ruby_test.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Test Hound
+class TestHound
+  NAME = 'Wormbots'
+  VERSION = '0.0.1'
+  LOGISTICS = [NAME, VERSION].freeze
+
+  def foo
+    puts 'bar'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.